.evergreen/install-node.sh

```diff
@@ -32,9 +32,27 @@ else
   [ -s "$NVM_DIR/bash_completion" ] && \. "$NVM_DIR/bash_completion"
 
   set +x # nvm is very verbose
-  echo nvm install --no-progress $NODE_JS_VERSION && nvm alias default $NODE_JS_VERSION
-  nvm install --no-progress $NODE_JS_VERSION
-  nvm alias default $NODE_JS_VERSION
+
+  # A few distros where pre-built node20 does not work out of the box and hence
+  # needs to be built from source
+  if [[ "${DISTRO_ID}" =~ ^(amazon|debian9|rhel7|suse|ubuntu18) ]];
```

Comment:
This matches newer Amazon versions like Amazon 2 and Amazon 2023 as well

Reply:
suse15 should also be fine to run as-is without compilation :)

Reply:
Ok, so I had previously only checked using docker, but you're right, I spawned a bunch of x64 suse linux hosts and checked their glibc versions :)

- suse12-small: 2.19
- suse12-sp5-small: 2.22
- suse15-small: 2.26
- suse15sp2-small: 2.26
- suse15sp4-small: 2.31

I think it would be fine to test Suse15 only using the SP4 variant, yes 👍

Reply:
Perfect. That I will add.

Reply:
Updated the regex here. I actually [tested all our amazon distros](https://spruce.mongodb.com/version/64c8fe863627e0e45398c751/tasks?sorts=STATUS%3AASC%3BBASE_STATUS%3ADESC) for regular install of node 20 and it turns out that only amazon2023* are the ones where a regular install works fine, good glibc, for the rest it fails hence I have included amazon-1* and amazon-2*. Since amazon2022* jobs never started, I assumed they are not available anymore hence also not included here.